dip-template/runtimes/dip-provider/src/lib.rs

```diff
@@ -746,3 +755,8 @@ impl_runtime_apis! {
 		}
 	}
 }
+
+cumulus_pallet_parachain_system::register_validate_block! {
```

Comment:
Can you move it up?

Reply:
moved back: [4e6c5e7](https://github.com/KILTprotocol/kilt-node/pull/627/commits/4e6c5e751155580b04f23c242903a0ae458deb8a)